src/unicodeMake.js

```diff
@@ -0,0 +1,59 @@
+/* eslint no-console:0 no-undef:0 */
+
+// This is an internal Node tool, not part of the KaTeX distribution,
+// whose purpose is to generate unicodeSymbols.js in this directory.
+// In this way, only this tool, and not the distribution/browser,
+// needs String's normalize function.
+
+const fs = require('fs');
+
+// Read supported accents from Parser.js
+const Parser = fs.readFileSync('./Parser.js', {encoding: 'utf8'});
+const match = /accents = [^;]*;/.exec(Parser);
+eval(match[0]);
+
+const encode = function(string) {
+    let output = '"';
+    for (let i = 0; i < string.length; i++) {
+        let hex = string.charCodeAt(i).toString(16);
+        while (hex.length < 4) {
+            hex = `0${hex}`;
+        }
+        output += `\\u${hex}`;
+    }
+    output = `${output}"`;
+    return output;
+};
+
+console.log("// This file is GENERATED by unicodeMake.js. DO NOT MODIFY.");
+console.log("");
+console.log("export const unicodeSymbols = {");
```

Comment:
Do we need this in addition to the `export default`?

Reply:
I'm still somewhat new to ES6, so I'm not sure what's normal here. I still find it weird that `export default const` is forbidden. I believe we could just do `export default {...}` if we don't need to name the object, so maybe that's best? The current system allows importing in two different ways (default or nondefault), but that's probably not necessary.

Reply:
I find the lack of support for `export default const` weird as well.  You can do:
```
export { foo as default };
```
I guess it's a moot point now since there no longer seems to be an `export default` in this file.